frontend/benefit/applicant/src/constants.ts

```diff
@@ -185,3 +187,9 @@ export enum VALIDATION_MESSAGE_KEYS {
 }
 
 export const dateRegex = /(?:(?:0?[1-9]|1\d|2\d)\.(?:0?[1-9]|1[0-2])|(?:30\.(?!02)(?:0[1-9]|1[0-2]))|(?:31\.(?:0[13578]|1[02])))\.(?:19|20)\d{2}/;
+export const addressRegex = /^([\d (),./A-Za-zÄÅÖäåö-]+)$/;
+export const companyAccRegex = /^FI\d{16}$/;
+export const phoneRegex = /^((\\+[1-9]{1,4}[ \\-]*)|(\\(\d{2,3}\\)[ \\-]*)|(\d{2,4})[ \\-]*)*?\d{3,4}?[ \\-]*\d{3,4}?$/;
+export const postalCodeRegex = /^\d{5}$/;
+export const namesRegex = /^[\w',.-][^\d!#$%&()*+/:;<=>?@[\\\]_{|}~¡¿÷ˆ]{2,}$/;
```

Comment:
This names regex seem to work quite well against these: https://www.w3.org/International/questions/qa-personal-names
Again some kyrydic russian names and japanese names does not work. The special character might crash the saving to database, should be asked about backend developers too what can be allowed. And also from PO too?

Reply:
good point. though i never had problems with saving russian chars with utf-8 encoding